### ISSUE TITLE
install failed for version 0.17.1

### ISSUE THREAD

**Issue by 308299160** [marktext/marktext#3324]
### Description

I get error while install the marktext(0.17.1)

![image](https://user-images.githubusercontent.com/12628451/174719781-18d2d4dd-ae36-46a1-9775-b7166e823d01.png)

### Steps to reproduce

1. install the version
2. open the software
3. prompt error

### Versions

- MarkText version: 0.17.1
- Operating system: macOs Monterey 12.4(M1)

**Comment by FlorianLoch** [comment]
Perhaps this version is more helpful:

<img width="265" alt="image" src="https://user-images.githubusercontent.com/5587805/176048448-4e31b789-1255-4ecf-bd85-861e12336c9e.png">

To me it seems like simply the arm64 binary got corrupted somewhere on the way.

**Comment by FlorianLoch** [comment]
Just to save others the trouble: have a look at this issue https://github.com/marktext/marktext/issues/2983

TL;DR: The binary isn't broken, it's just not signed. Run `xattr -r -d com.apple.quarantine /Applications/MarkText.app` to have macOS not put in in quarantine.

**Comment by fxha** [comment]
Thank you for reporting the issue. We noticed that this is a duplicate and closing it in favor of #2983. You may want to subscribe there for further updates.
